cosmo_tester/config_schemas/cfy_manager_upgrade.yaml

```diff
@@ -1,7 +1,7 @@
 namespace: upgrade
 upgrade_version:
   description: The version to upgrade to.
-  default: '5.1.2'
+  default: '5.3.0'
```

Comment:
This should stay `5.1.2`, as it tests the command `cfy_manager upgrade` and this command cannot be used with `5.3.0`.